CHANGELOG.md

```diff
@@ -4,6 +4,50 @@ All notable changes to this project will be documented in this file.
+
+**Implemented enhancements:**
+
+- Add parameter to manage the content of log4j2.properties [\#1260](https://github.com/voxpupuli/puppet-elasticsearch/pull/1260) ([h-haaks](https://github.com/h-haaks))
+- Add SLES 15 [\#1254](https://github.com/voxpupuli/puppet-elasticsearch/pull/1254) ([thomas-merz](https://github.com/thomas-merz))
+- Allow puppetlabs/java 11.x [\#1253](https://github.com/voxpupuli/puppet-elasticsearch/pull/1253) ([h-haaks](https://github.com/h-haaks))
+- Allow puppet/elastic\_stack 10.x [\#1252](https://github.com/voxpupuli/puppet-elasticsearch/pull/1252) ([h-haaks](https://github.com/h-haaks))
+- Add Ubuntu 24.04 [\#1251](https://github.com/voxpupuli/puppet-elasticsearch/pull/1251) ([h-haaks](https://github.com/h-haaks))
+- Add Debian 12 [\#1250](https://github.com/voxpupuli/puppet-elasticsearch/pull/1250) ([h-haaks](https://github.com/h-haaks))
+- Add Rocky 8 and 9 [\#1249](https://github.com/voxpupuli/puppet-elasticsearch/pull/1249) ([h-haaks](https://github.com/h-haaks))
+- Add AlmaLinux 8 and 9 [\#1248](https://github.com/voxpupuli/puppet-elasticsearch/pull/1248) ([h-haaks](https://github.com/h-haaks))
+- Add RedHat/CentOS/OracleLinux 9 [\#1247](https://github.com/voxpupuli/puppet-elasticsearch/pull/1247) ([h-haaks](https://github.com/h-haaks))
+- metadata.json: Add OpenVox [\#1241](https://github.com/voxpupuli/puppet-elasticsearch/pull/1241) ([jstraw](https://github.com/jstraw))
+- Add a parameter to package mark hold [\#1232](https://github.com/voxpupuli/puppet-elasticsearch/pull/1232) ([phaedriel](https://github.com/phaedriel))
+- Change license endpoint [\#1231](https://github.com/voxpupuli/puppet-elasticsearch/pull/1231) ([phaedriel](https://github.com/phaedriel))
+
+**Closed issues:**
+
+- Issues on the elasticsearch\_component\_template.rb and  elasticsearch\_index\_template.rb [\#1222](https://github.com/voxpupuli/puppet-elasticsearch/issues/1222)
+- why was log4j2.properties.erb configuration commented ? [\#1200](https://github.com/voxpupuli/puppet-elasticsearch/issues/1200)
+-  Elasticsearch API responded with: no handler found for uri \[/\_xpack/license?acknowledge=true\] and method \[PUT\] [\#1189](https://github.com/voxpupuli/puppet-elasticsearch/issues/1189)
+- elasticsearch\_keystore resource fails due to legacy instance code [\#1153](https://github.com/voxpupuli/puppet-elasticsearch/issues/1153)
+- elasticsearch\_keystore can't modify frozen String [\#1152](https://github.com/voxpupuli/puppet-elasticsearch/issues/1152)
+- Package not set to status hold on Debian [\#1133](https://github.com/voxpupuli/puppet-elasticsearch/issues/1133)
+- ESJsonLayout as layout type [\#1056](https://github.com/voxpupuli/puppet-elasticsearch/issues/1056)
+- Same template for both logging.yml and log4j2.properties [\#907](https://github.com/voxpupuli/puppet-elasticsearch/issues/907)
+- log4j config, how to add custom appender [\#529](https://github.com/voxpupuli/puppet-elasticsearch/issues/529)
```

Comment:
I added skip-changelog to those, because I don't think they provide a value to users in the CHANGELOG.md

Reply:
Agree